tests/unit/CMakeLists.txt

```diff
@@ -55,3 +55,7 @@ target_link_libraries(test_bruteforce PUBLIC gtest_main VectorSimilarity)
 include(GoogleTest)
 gtest_discover_tests(test_hnswlib)
 gtest_discover_tests(test_bruteforce)
+
+find_package(benchmark REQUIRED)
```

Comment:
where is the benchmark package coming from?

Reply:
Should be a dependency if we'll decide using google-benchmark